eos/rare-b-decays/lambda-b-to-lambda1520-ll-naive.cc

```diff
@@ -111,17 +121,25 @@ namespace eos
             HplusT5m32 = (m_Lb - m_Lstar) * sqrt(s_minus) * form_factors->f_perp32_t5(s);
 
         result.b_perp1_right =   sqrt(2.0) * norm_s * (wilson910_plus_right  * HplusVm32 - 2.0 * m_b_msbar / s * wilson7_plus  * HplusTm32);
-        result.b_perp1_left =    sqrt(2.0) * norm_s * (wilson910_plus_left   * HplusVm32 - 2.0 * m_b_msbar / s * wilson7_plus  * HplusTm32);
+        result.b_perp1_left  =   sqrt(2.0) * norm_s * (wilson910_plus_left   * HplusVm32 - 2.0 * m_b_msbar / s * wilson7_plus  * HplusTm32);
```

Comment:
I think this is now inconsistent. The quark masses should enter here either as ``m_b_msbar + m_s_msbar`` or ``m_b_msbar - m_s_msbar``. You are using the ``m_s_msbar -> 0`` limit here, but not below.

Reply:
As discussed offline this is correct due to the definition of O7